daemon/daemon.go

```diff
@@ -1113,11 +1117,10 @@ func NewDaemon(ctx context.Context, config *config.Config, pluginStore *plugin.S
 		case "windowsfilter":
 			// Docker WCOW graphdriver
 		case "":
-			// Use graph driver but enable migration
+			// Use graph driver but enable migration, unless opted-in
```

Comment:
this sentence is a bit hard to grasp; "enable migration, unless opted-in", or "use graph driver, unless opted-in"?